release_notes.md

```diff
@@ -267,141 +271,161 @@ This release is not required any additional steps to use.
+v1.1.0.
 
 1. [I] The new version of grpc has been added.  
-   [M] If you require the rdp v6 version of the interface, you do not need to do anything. 
-       Otherwise, you need to reinstall th2-grpc-data-provider lib to the required one for your rdp.
+   [M] If you require the rdp v6 version of the interface, you do not need to do anything.
+   Otherwise, you need to reinstall th2-grpc-data-provider lib to the required one for your rdp.
 
 More detail in [here](https://github.com/th2-net/th2-data-services/tree/dev_1.2.0#grpc-provider-warning)
 
 ## Features
+
 1. [TH2-3083] The problem with several versions of the grpc interface is solved.
 2. [TH2-3512] Provider V6 module is developed.
 3. [TH2-4141] Option to disable ssl certificate for rdp5 is added.
 4. [TH2-4098] Added Streams class for the param 'stream'.
 
 # BugFixes
+
 1. [TH2-4072] Now ETC doesn't raise a warning for missing detached_events.
 2. GRPC requests (start_timestamp, end_timestamp) are now made in UTC.
 
 # v1.2.1
 
 # BugFixes
+
 1. Added missing library importlib_metadata.
 
 # v1.2.2
 
 ## BugFixes
+
 1. [TH2-4195] EventsTree without parent raises `EventIdNotInTree` exception when trying to use `get_parent()` method.
 
 # v1.2.3
 
 ## BugFixes
 
-1. [TH2-4234] The library can now be run on Windows. 
+1. [TH2-4234] The library can now be run on Windows.
 
 # v1.3.0
 
 ## User impact and migration instructions
-This release implements performance bug fixes and provides Data object cache file saving and loading. 
 
-1. [I] Logging were removed from library. Only special builds will have logging. 
-      User cannot use `add_stderr_logger` and  `add_file_logger` logging functions.
+This release implements performance bug fixes and provides Data object cache file saving and loading.
+
+1. [I] Logging were removed from library. Only special builds will have logging.
+   User cannot use `add_stderr_logger` and  `add_file_logger` logging functions.
    [M] Remove DS lib logging usage anywhere.
 2. [I] Since `v1.3.0`, the library doesn't provide data source dependencies.
-   
-   [M] You should provide it manually during installation. 
-      You just need to add square brackets after library name and put dependency name.
+
+   [M] You should provide it manually during installation.
+   You just need to add square brackets after library name and put dependency name.
 
       ```
       pip install th2-data-services[dependency_name]
       ```
 
-      **Dependencies list** 
+   **Dependencies list**
 
-      | dependency name | provider version |
-      |:--------:|:-------:|
-      |   RDP5   |    5    |
-      |   RDP6   |    6    |
+| dependency name | provider version |
+|:---------------:|:----------------:|
+|      RDP5       |        5         |
+|      RDP6       |        6         |
 
-      **Example**
+**Example**
 
-      ```
-      pip install th2-data-services[rdp5]
-      ```
+   ```
+   pip install th2-data-services[rdp5]
+   ```
```

Comment:
Were all these changes made by some auto script, or you just solved to change it?

Reply:
Except table format (which was a little broken) pre-commit formatted it